internal/k8s/controller.go

```diff
@@ -1262,6 +1262,8 @@ func (c *AviController) Start(stopCh <-chan struct{}) {
 		informersList = append(informersList, lib.AKOControlConfig().AdvL4Informers().GatewayClassInformer.Informer().HasSynced)
 		go lib.AKOControlConfig().AdvL4Informers().GatewayInformer.Informer().Run(stopCh)
 		informersList = append(informersList, lib.AKOControlConfig().AdvL4Informers().GatewayInformer.Informer().HasSynced)
+		go c.dynamicInformers.VCFNetworkInfoInformer.Informer().Run(stopCh)
```

Comment:
should it be with `if IsVCFCluster()`  condition?

Reply:
yes, thanks

Reply:
yes, thanks